### PR TITLE
Todo app

### PR DESCRIPTION
**Pull request by yesi-aracawa**
Add test in some models with rspec

use in some models gem should-test

### CODE REVIEW

**Comment by Taniuchis** [comment]
Hi Yesi I think that you could add some tests for the controllers

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+    TodoList.new(title: "Anything", description: "Lorem ipsum",
```
Comment (Taniuchis) [inline spec/models/user_spec.rb]: You could add FactoryBot for your tests

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+    TodoList.new(title: "Anything", description: "Lorem ipsum",
+        created_at: DateTime.now, updated_at: DateTime.now + 1.week, user_id: 8)
+    subject {
+    User.new(:id => 8, :email => "yesi@mail.com", :password => "pw1234",
```
Comment (Taniuchis) [inline spec/models/user_spec.rb]: You could add Factorybot for your tests

File: spec/models/user_spec.rb (hunk 1 above)
Comment (yesi-aracawa) [reply spec/models/user_spec.rb]: thanks for your comment, but in some case it was not necesary, but for many tests i will concider, thaks ;) :D

**Pull request merged**